src/main/common/vector.h

```diff
@@ -57,6 +57,7 @@ float vector2Cross(const vector2_t *a, const vector2_t *b);
 float vector2NormSq(const vector2_t *v);
 float vector2Norm(const vector2_t *v);
 vector2_t *vector2Normalize(vector2_t *result, const vector2_t *v);
+vector2_t* vector2Rotate(vector2_t *result, const vector2_t *v, const float angle);
```

Comment:
```suggestion
vector2_t *vector2Rotate(vector2_t *result, const vector2_t *v, const float angle);
```

Reply:
Yes PL suggested it this way, but for consistency with other vector nomenclature I'll go with your suggestion.

Reply:
sorry, I usually use 'type* name' in other projects and sometimes mix it up

Reply:
No problem, as you can tell, I bend the way the wind blows when it comes to things like this :-)